Added classes to create tables  for EventCatalogAPI

LeelaCreations opened the pull request.

LeelaCreations committed:
Added classes to create tables  for EventCatalogAPI

sstambat12 commented on EventCatalog/Domain/EventsCatalog.cs:
```diff
@@ -0,0 +1,33 @@
+        public virtual Location Location { get; set; }
+
+        //ForiegnKey relation with Organizer table
+        public int OrganizerId { get; set; }
```
I think, It should be Property of an Event. If we create new organizer for each Event, it won't be Useful. We can discuss more when we will meet.

sstambat12 committed:
Created classes for EventCatalogAPI.

LeelaCreations committed:
removed the Organizer class and added Organizer name and description props under EventsCatalog class

LeelaCreations committed:
Added wwwroot folder and 20 event pics

LeelaCreations committed:
Added Seed class

LeelaCreations committed:
Added Location table and also added FK in Events Table

Commit:
Merge remote-tracking branch 'origin/leela' into sujata

Commit:
Fixing compilation errors after merging from leela

After I merged changes from leela branch, I got some compilation errors. Most of them were related to namespace change to EventCatalogAPI from EventCatalog.

Commit:
Created Pics Folder

Created New folder Pics under wwwroot and moved images in it.

Commit:
Added PicController API to get Images from wwwroot

Commit:
Seeding Database

Called CatalogSeed's seed method to seed the database.

Commit:
Removing Unnecessary Migration files.

Commit:
Added CatalogController API

Added few APIs in it to get all the events, get all the event types, event categories and event locations. And one API to filter events according to type, category and location

Commit:
Added Swagger to create documentation.

LeelaCreations committed:
testing the latest merged code from sujata
